Sds.Osdr.WebApi.IntegrationTests/Tests/CategoryEntities/AddCategoriesToEntity.cs

```diff
@@ -0,0 +1,90 @@
+﻿using FluentAssertions;
+using Leanda.Categories.Domain.ValueObjects;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Sds.Osdr.IntegrationTests;
+using Sds.Osdr.IntegrationTests.FluentAssersions;
+using Sds.Osdr.IntegrationTests.Traits;
+using Sds.Osdr.WebApi.IntegrationTests.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Sds.Osdr.WebApi.IntegrationTests
+{
+    public class AddCategoriesToEntityFixture
+    {
+        public Guid CategoryId;
+        public Guid BlobId { get; set; }
+        public Guid FileId { get; set; }
+
```

Comment:
too mauch white spaces here, also 57-58 and 74-75 decrease number of new lines to one